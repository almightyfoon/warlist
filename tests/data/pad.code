c6501b_-0R5r5X5X7jj6e05W7kg89u7B5X2_dSiF___-0J8gc53WdH7y743G3Hch8McahX
