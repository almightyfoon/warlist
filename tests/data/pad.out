https://conflictchamber.com/#c6501b_-0R5r5X5X7jj6e05W7kg89u7B5X2_dSiF___-0J8gc53WdH7y743G3Hch8McahX

Mercenary Army - 75 / 75 points
[Theme] Llaelese Resistance

[Ashlynn 1] Ashlynn d'Elyse [+29]
 - Mangler [15]
 - Mangler [15]
Anastasia di Bray [3]
Eilish Garrity, the Occultist [0(5)]
Gastone Crosse [0(4)]
 - Vanguard [10]
Gorman di Wulfe, Rogue Alchemist [4]
Major Harrison Gibbs [4]
Ragman [4]
Rutger Shaw, Professional Adventurer [4]
 - Mangler [15]
Choir of Menoth (min) [4]
Flame Bringers (max) [17]
Thorn Gun Mages [9]

Mercenary Army - 75 / 75 points
[Theme] The Irregulars

[Fiona 1] Fiona the Black [+31]
 - Galleon [39]
Eiryss, Mage Hunter of Ios [0(7)]
Gobber Tinker [2]
Orin Midwinter, Rogue Inquisitor [0(5)]
Cylena Raefyll & Nyss Hunters (min) [12]
Idrian Skirmishers (max) [15]
 - Idrian Skirmisher Chieftain & Guide [0(5)]
Kayazy Eliminators [5]
Lady Aiyana & Master Holt [8]
The Commodore Cannon & Crew [7]
Hammerfall Siege Crawler [18]
